docs/docs/api/LayoutAnimations/EntryAnimations.md

```diff
@@ -0,0 +1,169 @@
+---
+id: entryAnimations
+title: Entering Animations
+sidebar_label: Entering Animations
+---
+
+## How to use predefined entering animation?
+
+### 1. Import chosen animation
+```js
+    import { Animation } from 'react-native-reanimated';
+```
+### 2. Choose Animated Component which entering you want to animate
+```js
+    <AnimatedComponent entering={Animation} >
+```
+### 3. Customize the animation
+    If you don't know what modifiers the animation provides then find your animation down below.
```

Comment:
"Different type of entering animations can be customized differently. For the complete list of option please refer to the paragraph specific to theparticulr animation type"??